jmeter-java-dsl/src/main/java/us/abstracta/jmeter/javadsl/http/DslCookieManager.java

```diff
@@ -33,10 +64,22 @@ public DslCookieManager disable() {
     return this;
   }
 
+  public DslCookieManager setClearCookiesBetweenIterations(boolean clear) {
```

Comment:
Can you add javadocs describing the method purpose, when to use it, maybe give examples of best practices or commmon use cases. And remember adding a @since annotation to it so users know since what version was this method available in the API.

Reply:
done